src/components/AWSLambdaPageTable/AWSLambdaPageTable.tsx

```diff
@@ -165,81 +165,72 @@ export const AWSLambdaTableView: FC<Props> = ({
   );
 };
 
-export const AWSLambdaPageTable = () => {
+export const AWSLambdaPageTable = ({ entity }: { entity: Entity }) => {
   const [page, setPage] = useState(0);
   const [pageSize, setPageSize] = useState(5);
   const [filteredRows, setFilteredRows] = useState<LambdaData[]>([]);
-  let entityCompoundName = useEntityCompoundName();
-  if (!entityCompoundName.name) {
-    // TODO(shmidt-i): remove when is fully integrated
-    // into the entity view
-    entityCompoundName = {
-      kind: 'Component',
-      name: 'backstage',
-      namespace: 'default',
-    };
-  }
+
   const {
     lambdaNames: lambdaAnnotationList,
     value: projectName,
     loading,
-  } = useServiceEntityAnnotations(entityCompoundName);
-  if (!entityCompoundName.name) {
-    entityCompoundName = {
-      kind: 'Component',
-      name: 'backstage',
-      namespace: 'default',
-    };
-  }
-  const [settings, dispatch] = useContext(AppContext);
+  } = useServiceEntityAnnotations(entity);
+
+  const [
+    {
+      awsAccessKeyId,
+      awsAccessKeySecret,
+      authMethod,
+      identityPoolId,
+      region,
+      showSettings,
+    },
+    dispatch,
+  ] = useContext(AppContext);
```

Comment:
I like to always wrap useContext with custom hook. That way you could have your custom logic there (for example error handling in case someone tries to use context outside of provider). But in this simple example it doesn't matter.